### DISCUSSION TITLE
Change security onion management interface after installation

### DISCUSSION THREAD

**Discussion by hadi-d99** [Security-Onion-Solutions/securityonion#13372] [2.4]
### Version

2.4.80

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

8

### RAM

64

### Storage for /

2TB

### Storage for /nsm

2TB

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

After finishing installation, how to change the management interface in sec onion as when i change the interface
i log in and show the old ip (you can access the web portal using https://<oldip>

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by dougburks** [comment]
Depending on what exactly you're trying to do, you might want to review the following sections of the documentation:
https://docs.securityonion.net/en/2.4/ip.html
https://docs.securityonion.net/en/2.4/url-base.html
